### ISSUE TITLE
gcloud cli commands

### ISSUE THREAD

**Issue by andrewhatfield** [NetAppDocs/astra#110]
Page: [Set up Google Cloud](https://docs.netapp.com/us-en/astra/get-started/set-up-google-cloud.html)

It would be great to have sections like this include the `gcloud` examples

For example

```gcloud components update
gcloud services enable servicenetworking.googleapis.com \
    --project=PROJECT_ID```

**Comment by netapp-mwallis** [comment]
@andrewhatfield, the docs do link to Google documentation with gcloud CLI examples currently.  Are you saying we should include these examples in our docs?  Only for account creation, or for any GCloud related setup tasks?

**Comment by andrewhatfield** [comment]
Yes, make it easy for people without forcing them to jump through hoops.

Give them complete copy and paste instructions including CLI and code

For everything they need to do

**Comment by netapp-mwallis** [comment]
Hi @andrewhatfield, understood. Unfortunately I was informed that we have a policy against copying content from other vendors, for several reasons:

    We have no control over the content, and it could change at any time and be out of date
    We don't have permission from Google to reuse their documentation, images, or script / code examples

Sorry about that. We'll have to settle for linking to the content, but I will make it clearer that the destination link includes CLI examples, and any other method they might wish to use.

**Comment by netapp-mwallis** [comment]
Hi @andrewhatfield, I sent you an email regarding this issue earlier.  After some content analysis, we're going to close this request for the following reasons:

1.	If we include only the gcloud command examples, some folks might not care about gcloud commands, preferring REST or CLI instead, which means the added content would be of no use to them – they would still need to link out to the Google documentation
2.	The link to Google docs provides access to all methods of performing the task in one place, which is better than we can do in our documentation - and it opens in a new browser tab for them which they can close when they are done
3.	Adding the requested content would make our topic longer and more difficult to parse, and could make things worse if the copied commands become out of date / inaccurate
4.	If we include commands for one task, we should be consistent and include commands for all Google-related tasks, which would lengthen the topic considerably (related to point #4)'

Thanks for your understanding.  Please reach out with any questions.
